hs_core/management/commands/fix_resourcefile_duplicates.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+
+"""
+Fix resource file duplicates
+
+This checks that there is only one ResourceFile for each iRods file
+
+* By default, prints errors on stdout.
+* Optional argument --log instead logs output to system log.
+"""
+
+from django.core.management.base import BaseCommand
+from django.db.models import Count
+from hs_core.models import ResourceFile
+
+
+class Command(BaseCommand):
+    help = "Fix ResourceFile duplicates."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--dryrun',
+            action='store_true',
+            dest='dryrun',
+            default=False,
+            help='Find duplicates but dont fix them',
+        )
+
+    def handle(self, *args, **options):
+        dry_run = options['dryrun']
+
+        dup_resource_files = ResourceFile.objects.values('resource_file', 'object_id') \
```

Comment:
I was wondering if there are files that have `content_type` other than content type for CompositeResource. We used to have many different resource types supporting files. If there are such files then we should delete those first before finding duplicates for per object_id and deleting those.

Reply:
Thanks will add a check for that.

Reply:
@pkdash I attempted this but perhaps didn't understand exactly what you were intending for the content_type check

Reply:
We have to first get the ContenType object for the CompositeResource model.
```
desired_content_type = ContentType.objects.get_for_model(CompositeResource)
non_conforming_files = ResourceFile.objects.exclude(content_type=desired_content_type)
```

Reply:
thanks went ahead and updated that